hw/ip/alert_handler/dv/alert_handler_sim_cfg.hjson

```diff
@@ -79,21 +79,18 @@
     {
       name: alert_handler_entropy
       uvm_test_seq: alert_handler_entropy_vseq
-      run_opts: ["+test_timeout_ns=20_000_000_000"]
-      run_opts: ["+zero_delays=1"]
+      run_opts: ["+test_timeout_ns=20_000_000_000", "+zero_delays=1"]
```

Comment:
Aah, yes. HJson parser silently drops the previous one if it encounters two dicts with the same key.  You could put each option on a new line with a comment explaining it if you want to (but it is not needed for simple switches like these).  
```suggestion
      run_opts: [// This test takes awfully long!
                 "+test_timeout_ns=20_000_000_000", 
                 // Make agents not inject any random delays. 
                 "+zero_delays=1"]
```

Reply:
Thanks so much for the explanation Sri. I will keep this in mind.